web/src/ui/nodes/node-card/on-demand/in-line.ts

```diff
@@ -72,15 +78,18 @@ export class UIInlineOnDemand extends ToggleChipMixin(UIBaseCard) {
 
     const toolTipStyles = css`
       &::part(body) {
-        --sl-tooltip-padding: 0;
-        --sl-tooltip-border-radius: 0;
-        --sl-tooltip-background-color: transparent;
-        --sl-tooltip-color: ${(colors['black'] ?? 'black') as string};
         --max-width: 24rem;
-
+        color: ${(colors['black'] ?? 'black') as string};
+        padding: 0;
+        border-radius: 0;
+        background-color: transparent;
         pointer-events: all;
       }
 
+      &::part(base__arrow) {
+        display: none;
+      }
```

Comment:
I've refactored these styles to standard css properties/values, as setting the custom shoelace properties was cascading down to any instance of the `sl-tooltip` below this, eg the execution buttons.